### PR TITLE
Comment out HO event exporter to turn off HO latency measurement

### PR DESCRIPTION
**Pull request by woojoong88**

### CODE REVIEW

**Comment by woojoong88** [comment]
Before lightweight exposer is ready, I comment out HO event exporter.

File: pkg/southbound/e2.go [hunk 1]
```diff
@@ -61,7 +61,8 @@ type HOEventMeasuredRIC struct {
 
 // ListHOEventMeasuredRIC is the list of HOEvent which is measured at ONOS RIC
 var ListHOEventMeasuredRIC []HOEventMeasuredRIC
```
Comment (SeanCondon) [inline pkg/southbound/e2.go]: the array can also be commented out

**Comment by woojoong88** [comment]
> e function `exposeHOEventInfo` in exported.go can be removed.

Yup. You are absolutely right. Let me clean up all codes for HO latency measurement.

**Comment by SeanCondon** [comment]
Thanks for the update @woojoong88 - I think the exporter in the HO app and MLB app probably need to be handled too - maybe that can be a separate PR. I will merge this and other can be handled separately

**Pull request merged**